scripts/upload_code.py

```
@@ -0,0 +1,276 @@
+        if result.returncode != 0:
+            print(f"❌ Build failed:\n{result.stderr}")
+            return False
+            
+        print("✅ Build successful")
+        return True
+    
+    def upload_contract(self, contract: str, network: str, suri: str) -> Optional[str]:
+        """Upload contract code and return code hash"""
+        print(f"\n📤 Uploading contract code to {network}...")
+        
+        wasm_path = f"{contract}/target/ink/{contract}.wasm"
```

Flake8 (7.2.0)

[error] local variable 'wasm_path' is assigned to but never used

(F841)